website/docs/getting-started/setup/installation-guides/docker/install-business-edition.mdx

```diff
@@ -0,0 +1,17 @@
+# Docker
```

Comment:
```suggestion
# Deploy on Docker
```

Reply:
@pranayagarwal96 - One query - Do you think a header is needed in docs? Or the header will be added to the customer portal, and only steps are needed. Please confirm.

Reply:
Actually we don't need a header. Btw, you can also take a look at the DP of [customer portal](release-customer.appsmith.com) to get a sense of look & feel of the 'License & Installation' page where this will be embedded

Reply:
Thanks @pranayagarwal96 - We will remove the headers from installation as well as upgrade docs and just have steps into it. The DP link doesn't work. Could you please verify and share again? 

Are the pages embedded into the DP?